app/views/components/authorised_account_search_company_name.scala.html

```diff
@@ -36,19 +41,38 @@
 }
 
 <dl class="govuk-summary-list govuk-summary-list--no-border govuk-!-margin-bottom-0">
- @companyName.map { name =>
-  <div class="govuk-summary-list__row">
-   <dt class="govuk-summary-list__key govuk-!-width-three-quarters govuk-!-padding-top-0 govuk-!-padding-bottom-0">
-    @messages("cf.search.authorities.result.company.name")</dt>
-   <dd class="govuk-!-width-three-quarters govuk-summary-list__value govuk-!-padding-top-0 govuk-!-padding-bottom-0">@name</dd>
-  </div>
- }
-
- <div class="govuk-summary-list__row">
-  <dt class="govuk-summary-list__key govuk-!-width-three-quarters govuk-!-padding-top-0 govuk-!-padding-bottom-0">
-   @messages("cf.search.authorities.result.eori.number")</dt>
-  <dd class="govuk-!-width-three-quarters govuk-summary-list__value govuk-!-padding-top-0 govuk-!-padding-bottom-0">@query</dd>
- </div>
+    @companyName.map { name =>
+    <div class="govuk-summary-list__row">
```

Comment:
Not a biggie however we should try avoid using HTML syntax and import the GOV.UK components, there is already a component for this https://design-system.service.gov.uk/components/summary-list/